content/tutorials/publish-subscribe.textile

```diff
@@ -45,7 +96,13 @@ blang[java].
   import io.ably.lib.realtime.AblyRealtime;
 
   public class ExampleActivity extends AppCompatActivity {
-    private final static String API_KEY = "INSERT-YOU-API-KEY-HERE"; 
+    private final static String API_KEY = "INSERT-YOUR-API-KEY-HERE"; 
+    /* RuntimeException will be thrown if API_KEY will not be set to a proper one */
+    static {
```

Comment:
Only needed in the tutorial code, not here in the tutorial step-through.  This code does not help the user in anyway